### PR TITLE
Port (CDAP-1395) Log rotation of containers to 2.6

### PR DESCRIPTION
**Pull request by ParthGandhi**
Had a couple of merge conflicts that I had to fix - mostly around namespaces.

Still need to test against 2.6, but waiting for Albert's beamer PR to get merged.

### CODE REVIEW

**Comment by ParthGandhi** [comment]
http://builds.cask.co/browse/CDAP-DUT1080-1

**Comment by wolf31o2** [comment]
You can always spin up a CDAP 2.6 cluster using Coopr by setting:
```json
{"cdap":{"version":"2.6.1-1"}}
```
the_more_you_know_banner

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractDistributedProgramRunner.java [hunk 1]
```diff
@@ -104,7 +108,35 @@ public final ProgramController run(final Program program, final ProgramOptions o
...
+            LOG.error("Error copying container logback.", e);
+          }
+        }
+      }
+    } catch (URISyntaxException e) {
+      LOG.error("Error getting logback for program.", e);
+    }
+
+    final URI programLogbackURI = logbackURI;
+    final String programOptions = GSON.toJson(options);
```
Comment (poornachandra) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractDistributedProgramRunner.java]: This looks like unintended change.

**Comment by poornachandra** [comment]
One comment, otherwise LGTM :+1:

**Comment by ParthGandhi** [comment]
http://builds.cask.co/browse/CDAP-DUT1080-3

**Pull request merged**